junifer/data/masks/_ants_mask_warper.py

```diff
@@ -83,6 +88,17 @@ def warp(
         if dst == "T1w":
```

Comment:
why is this T1w here?

Reply:
Because that's provided here: https://github.com/juaml/junifer/blob/2d974e1f1c9c569e5f55f2d7e61460f526d931f0/junifer/data/masks/_masks.py#L536

Reply:
I meant that it does not make any sense to asume that T1w is native space.

The comment says "native space warping" and then check if `dst = "T1w"`. This would be true if the space of the T1w is native.

Reply:
Fair enough.